src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -66,6 +66,7 @@
       TODO: add OptimizeAssemblies="lib/netstandard2.0/System.Threading.Tasks.Extensions.dll"
     -->
     <ExpectedDependency Include="System.Threading.Tasks.Extensions" />
+    <ExpectedDependency Include="Microsoft.Bcl.AsyncInterfaces"/>
```

Comment:
I don't see this being deployed anywhere, what is carrying it along? If you run this code right now, where is VS loading the assembly from?

Reply:
talking to @AArnott it sounds like this is ok.  VS already has this dep, so we should just pick that up, right?

Reply:
Confirmed this with @AArnott on the main thread, so we're good now.